components/brave_wallet/browser/solana_keyring_unittest.cc

```diff
@@ -95,7 +95,8 @@ TEST(SolanaKeyringUnitTest, ImportAccount) {
 
   std::vector<uint8_t> private_key;
   ASSERT_TRUE(base::HexStringToBytes(
-      "2b4be7f19ee27bbf30c667b642d5f4aa69fd169872f8fc3059c08ebae2eb19e7",
+      "2b4be7f19ee27bbf30c667b642d5f4aa69fd169872f8fc3059c08ebae2eb19e7"
+      "a4b2856bfec510abab89753fac1ac0e1112364e7d250545963f135f2a33188ed",
```

Comment:
Is this parameter change because we're now concatenating public/private keys rather than just the pubkey or something like that?

Reply:
Yes. 
https://cyphr.me/ed25519_tool/ed.html
![image](https://github.com/user-attachments/assets/d255a963-f672-4b30-87a7-233793b76676)